site/en/docs/extensions/mv3/devtools/index.md

```diff
@@ -192,18 +170,24 @@ chrome.runtime.onConnect.addListener(function(devToolsConnection) {
 });
 ```
 
-### Evaluating JavaScript in the inspected window {: #evaluating-js }
+If a content script has already been injected, you can add additional context scripts using the
+`eval()` method. See [Pass the Selected Element to a Content Script][header-selected-element] for
+more information.
+
+### Evaluate JavaScript in the inspected window {: #evaluating-js }
 
 You can use the [`inspectedWindow.eval()`][api-inspectedwindow-eval] method to execute JavaScript
 code in the context of the inspected page. You can invoke the `eval()` method from a DevTools page,
-panel or sidebar pane.
+panel, or sidebar pane.
 
-By default, the expression is evaluated in the context of the main frame of the page. Now, you may
-be familiar with the DevTools [Console Utilities API][doc-utilities] features like element
-inspection (`inspect(elem)`), breaking on functions (`debug(fn)`), copying to clipboard (`copy()`)
-and more. `inspectedWindow.eval()` uses the same script execution context and options as the code
-typed at the DevTools console, which allows access to these APIs within the eval. For example,
-[SOAK][gh-soak] uses it for inspecting an element:
+Caution: Use `eval()` only if you need access to the JavaScript content of an inspected page. Otherwise,
+we recommend using the [`scripting.executeScript()`][api-scripting-executescript] method run scripts. For
```

Comment:
```suggestion
we recommend using [`scripting.executeScript()`][api-scripting-executescript] to run scripts. For
```

Reply:
This is legitimately a caution. Please make it [an aside](https://web.dev/handbook/web-dev-components/#caution-asides). FYI, the word 'caution' will be supplied by the aside so please remove it.

Reply:
Got it. For what it's worth, the Dev Docs style guide does it [this way](https://developers.google.com/devsite/reference/styles/notices#caution), and I didn't realize you've implemented it differently.